cpp/velox/compute/VeloxInitializer.cc

```diff
@@ -40,29 +41,95 @@ DECLARE_int32(split_preload_per_driver);
 
 using namespace facebook;
 
-namespace gluten {
+namespace {
+
+const std::string kHiveConnectorId = "test-hive";
+const std::string kVeloxCacheEnabled = "spark.gluten.sql.columnar.backend.velox.cacheEnabled";
+
+// memory cache
+const std::string kVeloxMemCacheSize = "spark.gluten.sql.columnar.backend.velox.memCacheSize";
+const std::string kVeloxMemCacheSizeDefault = "1073741824";
+
+// ssd cache
+const std::string kVeloxSsdCacheSize = "spark.gluten.sql.columnar.backend.velox.ssdCacheSize";
+const std::string kVeloxSsdCacheSizeDefault = "1073741824";
+const std::string kVeloxSsdCachePath = "spark.gluten.sql.columnar.backend.velox.ssdCachePath";
+const std::string kVeloxSsdCachePathDefault = "/tmp/";
```

Comment:
not related to the PR. 
We mentioned to change the ssd cache from /tmp/ to the shuffle dirs as spark_local_dirs. Do we have an issue to track this? If not, let's create one